tests/foreman/api/test_contentview.py

```diff
@@ -185,6 +186,31 @@ def test_positive_add_custom_content(self):
         self.assertEqual(len(content_view.repository), 1)
         self.assertEqual(content_view.repository[0].read().name, yum_repo.name)
 
+    @tier2
+    @run_only_on('sat')
+    def test_positive_add_custom_module_streams(self):
+        """Associate custom content (module streams) in a view
+
+        :id: 9e4821cb-293a-4d84-bd1f-bb9fff36b143
+
+        :expectedresults: Custom content (module streams) assigned and
+        present in content view
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        yum_repo = entities.Repository(product=product,
```

Comment:
that is more appropriate format
```
yum_repo = entities.Repository(
    product=product,
    url=CUSTOM_MODULE_STREAM_REPO_2
).create()
```

Reply:
please look closer on provided example by me.
Of course, it is not a blocker and it is not important, just for consistency